src/MusicStore/config.json

```diff
@@ -1,4 +1,7 @@
 ﻿{
+    "ApplicationInsights": {
+        "InstrumentationKey": "797d1481-9988-48f2-b41b-f4e305cd9207"
+    },
     "DefaultAdminUsername": "Administrator@test.com",
     "DefaultAdminPassword": "YouShouldChangeThisPassword1!",
```

Comment:
Good point, although we don't consider the Application Insights instrumentation keys to be secrets because they are embedded in plain text in the JavaScript instrumentation code and in the telemetry that can be sent by HTML clients over HTTP. What would be a proper way to store the admin user name and password values in this example? Can we do the same for the instrumentation key?